appscale/tools/utils.py

```diff
@@ -91,7 +73,7 @@ def config_from_dir(file_name, source_path):
 
   shortest_path = candidates[0]
   for candidate in candidates:
-    if path_length(candidate) < path_length(shortest_path):
+    if len(candidate.split(os.sep)) < len(shortest_path.split(os.sep)):
       shortest_path = candidate
```

Comment:
Wouldn't that just give me the length of the shortest path and not the shortest path itself?